app/assets/javascripts/discourse/components/image-uploader.js.es6

```diff
@@ -1,9 +1,39 @@
-import computed from "ember-addons/ember-computed-decorators";
+import {
+  default as computed,
+  observes
+} from "ember-addons/ember-computed-decorators";
+
 import UploadMixin from "discourse/mixins/upload";
+import lightbox from "discourse/lib/lightbox";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 
 export default Ember.Component.extend(UploadMixin, {
   classNames: ["image-uploader"],
-  infoHidden: true,
+
+  init() {
+    this._super(...arguments);
+    this._applyLightbox();
+  },
+
+  willDestroyElement() {
+    this._super(...arguments);
+    $("a.lightbox").magnificPopup("close");
+  },
+
+  @observes("imageUrl")
```

Comment:
We are trying to get away from observers unless completely necessary. Is there any other way this method can be triggered? @jjaffeux might have some ideas.

Reply:
Actually, I can apply the lightbox when the upload is done. Fixed :)